.gitignore

```diff
@@ -7,7 +7,6 @@ dist
 *.log
 coverage
 .env
-package-lock.json
```

Comment:
why are we bringing back this file?

Reply:
> package-lock.json is now comitted to have controlled deps for npm

same reason we have `yarn.lock`

Reply:
We use yarn.lock for predictable building in travis, I think keeping the 2 of them is confusing...